modules/UI/videolayout/VideoLayout.js

```diff
@@ -348,6 +345,27 @@ var VideoLayout = {
         if (remoteVideo) {
             remoteVideo.removeRemoteStreamElement(stream);
         }
+        this.updateMutedForNoTracks(id, stream.getType());
+    },
+
+    /**
+     * If participant has no tracks will make the UI display muted status.
+     * @param {string} participantId
+     * @param {string} mediaType 'audio' or 'video'
+     */
+    updateMutedForNoTracks(participantId, mediaType) {
+        const participant = APP.conference.getParticipantById(participantId);
+
+        if (participant
+                && !participant.getTracksByMediaType(mediaType).length) {
+            if (mediaType === 'audio') {
+                APP.UI.setAudioMuted(participantId, true);
+            } else if (mediaType === 'video') {
+                APP.UI.setVideoMuted(participantId, true);
```

Comment:
I guess no... the APP.UI.setAudio(Video)Muted is still used or am I missing something ?